godet.go

```diff
@@ -1013,6 +1013,23 @@ func (remote *RemoteDebugger) GetAllCookies() ([]Cookie, error) {
 	return cookies.Cookies, nil
 }
 
+//Set browser cookies  
+func (remote *RemoteDebugger) SetCookies(cookies []Cookie)error {
+	params := Params{}
+	if len(cookies) < 1 {
```

Comment:
Unless Network.setCookies fails if the array is empty, I wouldn't consider an error passing an empty array.